.jenkins/pipelines/Agnostic/Linux/Jenkinsfile

```diff
@@ -110,19 +109,16 @@ try{
     if(FULL_TEST_SUITE == "true") {
         stage("Full Test Suite") {
             testing_stages += [
-                "Sim 1804 clang-7 SGX1 Debug":      { simulationContainerTest('18.04', 'Debug',   ['-DHAS_QUOTE_PROVIDER=OFF', '-DLVI_MITIGATION=None', '-DLVI_MITIGATION_SKIP_TESTS=OFF']) },
-                "Sim 1804 clang-7 SGX1 Release":    { simulationContainerTest('18.04', 'Release', ['-DHAS_QUOTE_PROVIDER=OFF', '-DLVI_MITIGATION=None', '-DLVI_MITIGATION_SKIP_TESTS=OFF']) },
-                "Sim 1804 clang-7 SGX1FLC Debug":   { simulationContainerTest('18.04', 'Debug',   ['-DHAS_QUOTE_PROVIDER=ON',  '-DLVI_MITIGATION=None', '-DLVI_MITIGATION_SKIP_TESTS=OFF']) },
-                "Sim 1804 clang-7 SGX1FLC Release": { simulationContainerTest('18.04', 'Release', ['-DHAS_QUOTE_PROVIDER=ON',  '-DLVI_MITIGATION=None', '-DLVI_MITIGATION_SKIP_TESTS=OFF']) }
+                "Sim 1804 clang-7 SGX1 Debug":   { simulationContainerTest('18.04', 'Debug',   ['-DLVI_MITIGATION=None', '-DLVI_MITIGATION_SKIP_TESTS=OFF']) },
```

Comment:
Something kind of interesting is that if we look at the Jenkins files, it appears we no longer test SGX1FLC configurations and we only test SGX1, which the inverse is true.

Can you change the title of one of each test in this PR so they know we are testing SGX1FLC?

i.e.

Sim 1804 clang-7 **SGX1FLC** Release
Sim 1804 clang-7 **SGX1FLC** Debug

While functionally I understand they are the same, it might cause confusion.

Reply:
Good point. Thinking about it a bit more..should references to SGX be removed from Agnostic pipelines?
OE is built the same way irrespective of whether the build machine is SGX or not. And Agnostic pipelines don't run on SGX hardware.

Reply:
Honestly, I think these files are in need of a little TLC. 

Agnostic/ doesn't need "SGX" mentions when it refers to hardware, but it raises the question of if there should be an agnostic container with and without the SGX packages, would love your thoughts.

Azure/* doesn't need "ACC" mentions because it's all ACC